### ISSUE TITLE
Seeking advice on training DC-AE model

### ISSUE THREAD

**Issue by sonderlau** [lavinal712/AutoencoderKL#8]
Thank you very much for your open-source work on DC-AE, which has been extremely helpful for my research. 
I am a postgraduate student majoring in computer science, and my work focuses on using AI for weather forecasting.

However, while training my own dataset ([SEVIR](https://registry.opendata.aws/sevir/) ) I encountered some issues with the metrics and I would like to seek your advice. Although I am aware that you mentioned in other issues ( #5 ) that you don't have the resources and time to reproduce the metrics mentioned in the original paper, I would still like to ask you some questions about the training process.  

I am using the VIL data from the SEVIR dataset, which stands for Vertical Integrated Liquid, essentially precipitation data.

Model configs:

All the configurations were set exactly same as the provided yaml file, except: 
**in_channels = 12` , `latent_channels = 64**

The key metrics(in validation set) are as follows:

| Stage   | Epochs | Resolution | SSIM     | REC      | LPIPS    | PSNR      |
| ------- | ------ | ---------- | -------- | -------- | -------- | --------- |
| Phase 1 | 158    | 192*192    | 0.755699 | 0.041284 | 0.109648 | 35.197456 |
| Phase 2 | 60     | 384*384    | 0.650754 | 0.040414 | 0.114506 | 36.794658 |
| Phase 3 | 105    | 192*192    | 0.752387 | 0.041167 | 0.110347 | 35.422493 |

> I'll upload the metrics.csv files if you need more details.

Thoses metrics are computed only on DC-AE model, which means I didn't involve any Diffusion model.

The DC-AE focusing on transforming data into latent space (encoder), and the do the reversing process (decoder). So I think that metrics should be much higher than those above (such as SSIM should be almost around 0.9 or so, instead of 0.65~0.75 ). 

If these metrics do not achieve excellent results, I believe it will lead to very bad consequences: assuming that Diffusion model predicts the results perfectly, there will still be a loss of prediction accuracy when passing through the VAE decoder. This is unacceptable to me because my prediction task is data-sensitive, so I hope to achieve excellent numerical accuracy.

Thank you very much for your work. I look forward to your reply.

**Comment by sonderlau** [comment]
Thank you for your reply. I rethought it according to the advice you gave.

First, I compared the Metrics you mentioned and compared the training results of my model with those of some SOTA models. Judging from the results I have achieved so far, they are quite good.

I made a visual comparison of several VAE results again, especially by magnifying the local details. I think the restoration of these details still fails to meet my expectations. Then I went to learn about some of the work done using the Diffusion model for super-resolution tasks. I compared their results and found that they were not as expected either.

Summary: The current Diffusoin generative models are not yet good enough for numerically sensitive weather prediction tasks, which might exactly be the direction of my research.

No further questions, thank you.

Good luck with everything !
